geometry_msgs/Pose wp
float64 vel
duration dur
float64 tol_trans
float64 tol_angle
